### ISSUE TITLE
`forecast.mbl()` gains a new argument `names_to = ".model"`?

### ISSUE THREAD

**Issue by earowang** [tidyverts/fabletools#96]
hence we can easily rename the `.model` column.

**Comment by mitchelloharawild** [comment]
Sounds reasonable, although `".model"` is assumed in a variety of places (plotting, accuracy, etc.). It will probably need to be stored as an attribute of the `fbl_ts`

**Comment by earowang** [comment]
isn't it part of `key`?

**Comment by mitchelloharawild** [comment]
Yes, but `".model"` is treated differently from key in some places. For example when plotting, the model is used for the colour, other keys are row facets, and response variable (if multivariate) are column facets.

**Comment by earowang** [comment]
can `setdiff()` work out the difference?

**Comment by mitchelloharawild** [comment]
Difference from what? The original tsibble keys aren't stored, just the current ones (inclusive of .model).
Perhaps in the plot it could be used as the data may be available, although it is possible to plot the forecasts without data.

**Comment by mitchelloharawild** [comment]
`.model` is also used with `augment()`, `glance()`, `tidy()`, etc.
Should all these functions gain this argument?

**Comment by earowang** [comment]
I think so, whenever to "longer" format.

Can we just have `model` instead of `mdl_nm`?

**Comment by mitchelloharawild** [comment]
Consolidated into #192

**Comment by mitchelloharawild** [comment]
Reopening as I don't have time to include this in today's release, which closes #192.
